lab_1_classify_by_unigrams/main.py

```diff
@@ -34,6 +44,27 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if tokens is None or not isinstance(tokens, list):
+        return None
+
+    freq_dict = {}
+    new_tokens = []
+
+    for token in tokens:
+        if isinstance(token, str) and len(token) == 1:
```

Comment:
return None here if the conditions are opposite

Reply:
done